gradio/components/chatbot.py

```diff
@@ -84,6 +84,13 @@ class Message(GradioModel):
     content: Union[str, FileMessage, ComponentMessage]
 
 
+class SuggestionMessage(TypedDict):
+    icon: NotRequired[str]
+    display_text: NotRequired[str]
+    text: str
```

Comment:
Should text be required? What if the suggestionmessage is just an image for example?